tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -347,6 +350,74 @@ void SetHtmlResponseContent()
             return blockingAction;
         }
 
+        private string GetJsonResponse()
+        {
+            try
+            {
+                if (_blockedJsonTemplateCache != null)
+                {
+                    return _blockedJsonTemplateCache;
+                }
+
+                if (_settings.BlockedJsonTemplatePath != null && AppDomain.CurrentDomain.BaseDirectory != null)
+                {
+                    var fullPath =
+                        Path.IsPathRooted(_settings.BlockedJsonTemplatePath)
+                            ? _settings.BlockedJsonTemplatePath
+                            : Path.Combine(AppDomain.CurrentDomain.BaseDirectory, _settings.BlockedJsonTemplatePath);
+                    if (File.Exists(fullPath))
```

Comment:
Should we log a warning if the customer has specified a path but it doesn't exist?

Reply:
fixed in https://github.com/DataDog/dd-trace-dotnet/commit/68156022677fa23f7162678e322e888f850ddb46